.golangci.yml

```diff
@@ -3,4 +3,12 @@ run:
   - e2e
   skip-dirs:
   - vendor
-  - pkg/client/clientset/versioned/fake
+  - pkg/client/clientset/(.*)/fake
+linters:
+  enable:
```

Comment:
Neat! Would be nice to have a common list of linters for all Tekton projects in plumbing

Reply:
Indeed. I didn't dig too much in how `golangci-lint` loads configurations (and if it can load multiple ones), but yeah :angel: